bitnami/wordpress/6/debian-11/rootfs/opt/bitnami/scripts/libwordpress.sh

```diff
@@ -226,6 +227,12 @@ wordpress_initialize() {
                 WORDPRESS_DATA_TO_PERSIST+=" ${htaccess_file}"
             fi
         fi
+    else
+        if is_boolean_yes "$WORDPRESS_HTACCESS_OVERRIDE_NONE"; then
+            local htaccess_file="${APACHE_HTACCESS_DIR}/wordpress-htaccess.conf"
+        else
+            local htaccess_file="${WORDPRESS_BASE_DIR}/.htaccess"
+        fi
```

Comment:
To me, it is confusing to add this part of the code here just so that `htaccess_file` is defined with the correct value. I would rather compute the right value inside the `wordpress_disable_xmlrpc_endpoint` function itself.

Reply:
Ok, fixed this.